doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -139,22 +418,36 @@ Requirements:
 
    #. ``GML Simple Features``: ``SF-0``
 
-   .. image:: tewfs-1_1.jpg
+   .. image:: ./image/tewfs-1_1.png
 
 Run WMS 1.1 tests
```

Comment:
Good thing that we have WMS 1.1, but we're missing WMS 1.3, please add it.

Reply:
solved with https://github.com/randomorder/geoserver/pull/3/commits/47a6f7ee076dfe52c579b644dffa7142d8e97d4c